hellomama_registration/settings.py

```diff
@@ -356,5 +356,7 @@
 V2N_FTP_PASS = os.environ.get('V2N_FTP_PASS')
 V2N_FTP_ROOT = os.environ.get('V2N_FTP_ROOT')
 
+DBLINK_CONN = ''
```

Comment:
Should this not get the value from `os.environ` so that we can specify it in our config?